readme.md

```diff
@@ -277,17 +277,17 @@ In order to access those values from Rails `params`, we just have to treat it li
 
```

Comment:
They already learned about where `params` values come from. This is a good opportunity for you to ask them as review, rather than explicitly state when delivering your lesson. Not necessary to change anything in the lesson plan, but a good way to deliver.